### PR TITLE
Add 22150, 23663, 21679 ez_code

### PR DESCRIPTION
**Pull request by w8385**
- [x] Did you stage index file?
- [x] Did you follow [README.md](https://github.com/kiwiyou/boj-user-translation/blob/main/README.md)?
- [x] Did you follow [formatting guide](https://github.com/kiwiyou/boj-user-translation/blob/main/formatting.md)?

### CODE REVIEW

**Comment by w8385** [comment]
# 22150
image

**Comment by w8385** [comment]
# 23663
image

**Comment by w8385** [comment]
# 21679
image

File: src/23663/ko_KR-ez_code.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+<section id="input">
+    <div class="headline">
+        <h2>입력</h2>
+    </div>
+    <div id="problem_input" class="problem-text">
+        <p>첫 줄에 테스트 케이스의 수 $T$ ($1 \le T \le 100$)가 주어집니다. 이어 $T$개의 테스트 케이스가 묘사됩니다.</p>
+        <p>각 테스트 케이스의 형식은 아래와 같습니다.</p>
+        <p>첫 줄에 각각 붉은 옷 노인과 흰 옷 노인의 더미 수를 나타내는 두 정수 $n$, $m$ ($1 \le n, m \le 100$)이 주어집니다.</p>
+        <p>다음 줄에 붉은 옷 노인의 검정 돌 더미에 있는 돌의 수를 나타내는 $n$개의 정수 $a_i$ ($1 \le ai \le 10 ^ 9$)가 주어집니다.</p>
+        <p>다음 줄에 흰 옷 노인의 흰색 돌 더미에 있는 돌의 수를 나타내는 $m$개의 정수 $b_i$ ($1 \le bi \le 10 ^ 9$)가 주어집니다.</p>
```
Comment (kiwiyou) [inline src/23663/ko_KR-ez_code.html:26]: 아래 첨자가 누락되었습니다.

**Review by kiwiyou — CHANGES_REQUESTED** [review]
번역 감사합니다! 수정사항을 반영해 주세요.

File: src/23663/ko_KR-ez_code.html (hunk 1 above)
Comment (w8385) [reply src/23663/ko_KR-ez_code.html]: [2923d2d](https://github.com/kiwiyou/boj-user-translation/pull/49/commits/2923d2d5d42d84de6353c6cc72249639510e3dbe) 수정했습니다!

**Review by kiwiyou — APPROVED** [review]

**Pull request merged**